specs/Autofill.md

```diff
@@ -0,0 +1,104 @@
+# Background
+The WebView2 team has been asked for an API to allow end developers to set the autofill preferences.  We are exposing two of the autofill preferences that will allow enabling and disabling general and password autofill.  General autofill includes things like email addresses, shipping addresses, phone numbers, and names.  Password autofill includes things like usernames and passwords for login. 
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+
+# Description
+
+Autofill has three components
+* Auto-populate - Populate the corresponding form fields automatically on page load.
+* Suggest - When the user clicks on the form field, drop down suggestions of previously saved forms will be displayed.
+* Populate - When clicking on one of the suggestions, the form data will populate the respective fields.
+
+
+# Examples
+
+## Win32 C++
+```cpp
+void SettingsComponent::TogglePasswordAutofill() {
+    wil::com_ptr<ICoreWebView2Settings2_2> settings;
+    webView->get_Settings(&settings);
+    bool enabled;
+    settings->get_IsPasswordAutofillEnabled(&enabled);
+    settings->put_IsPasswordAutofillEnabled(!enabled);
+}
+
+void SettingsComponent::ToggleGeneralAutofill() {
+    wil::com_ptr<ICoreWebView2Settings2_2> settings;
+    webView->get_Settings(&settings);
+    bool enabled;
+    settings->get_IsGeneralAutofillEnabled(&enabled);
+    settings->put_IsGeneralAutofillEnabled(!enabled);
+}
+```
+
+## .NET, WinRT
+```c#
+webView2Control.NavigationStarting += TogglePasswordAutofill;
```

Comment:
This doesn't seem like a real scenario. We wouldn't just toggle autofill back and forth on every navigation. Ideally the sample demonstrates real usage of this. You could imagine that there's a menu item or button that calls the Toggle...Autofill method and you don't hook it up to the NavStarting event, maybe that's enough?